A.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: brown;
+}
```

Comment:
- [OPTIONAL] Please remember that the commit messages should be descriptive, written in the imperative tense, and capitalized. For example in this case could be:  "Add xxxx.css file."  :nerd_face: